test/rulesets/Base/arraymath.jl

```diff
@@ -49,6 +49,43 @@
     end
 
 
+    VERSION >= v"1.6.0-DEV.1536" && @testset "muladd: $T" for T in (Float64, ComplexF64)
+        @testset "add $(typeof(z))" for z in [rand(T), rand(T,3), rand(T,3,3)] #, false]
+            dz = rand(T, fill(3, ndims(z))...)
+            @testset "matrix * matrix" begin
+                A, B = rand(T,3,3), rand(T,3,3)
```

Comment:
BlueStyle prescribed spaces after commas
can you do ctrl+F `rand(T,3,3)` to `rand(T, 3, 3)`
or run JuliaFormatter set to BlueStyle on this file?

Reply:
There is also an open issue discouraging multiple assignements on  single like
https://github.com/invenia/BlueStyle/issues/80

Reply:
Done.